clang/test/SemaHLSL/BuiltIns/dot-errors.hlsl

```diff
@@ -22,7 +22,7 @@ float test_dot_vector_size_mismatch(float3 p0, float2 p1) {
 
 float test_dot_builtin_vector_size_mismatch(float3 p0, float2 p1) {
   return __builtin_hlsl_dot(p0, p1);
-  // expected-error@-1 {{all arguments to '__builtin_hlsl_dot' must have vectors of the same type}}
+  // expected-error@-1 {{all arguments to '__builtin_hlsl_dot' must have the same type}}

```

Comment:
I don't think this error ever fires on non-vectors. I find the "vectors of the same type" wording to be clearer, but I don't feel strongly about it.

Reply:
it is but consolidating to one diagnostic means changing the language here isn't an option anymore without having to update a bunch of other projects Sema error tests.